tests/docker/test_filter_deep_crawl.py

```
@@ -0,0 +1,201 @@
+    
+    try:
+        async with httpx.AsyncClient() as client:
+            print("\n1. Sending crawl request to REST API...")
+            response = await client.post(
+                f"{BASE_URL}crawl",
+                json=crawl_payload,
+                timeout=30
+            )
+            
+            if response.status_code == 200:
+                print(f"✅ REST API returned 200 OK")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)